src/content/docs/de/guides/upgrade-to/v3.mdx

```diff
@@ -0,0 +1,679 @@
+
+### Standardwert Geändert: `scopedStyleStrategy`
+
+In Astro v2.x war der Standardwert von [`scopedStyleStrategy`](/de/reference/configuration-reference/#scopedstylestrategy) `"where"`.
+
+Astro v3.0 führt einen neuen Standardwert ein: `"attribute"`. Standardmäßig werden Styles jetzt mit `data-*`-Attributen angewendet.
+
+#### Was soll ich tun?
+
+Um das aktuelle [Style-Scoping](/de/guides/styling/#scoped-styles) deines Projekts beizubehalten, aktualisiere die Konfigurationsdatei auf den vorherigen Standardwert:
+
+
+```js title="astro.config.mjs" ins={4}
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+  scopedStyleStrategy: "where"
+})
+```
+
+### Standardwert Geändert: `inlineStyleSheets`
+
+In Astro v2.x wurden standardmäßig alle Projekt-Styles als Link-Tags gesendet. Du konntest wählen, sie jedes Mal in `<style>`-Tags einzufügen, indem du `"always"` verwendest, oder nur Styles unter einer bestimmten Größe mit `"auto"` inlining durch die Konfiguration [`build.inlineStylesheets`](/de/reference/configuration-reference/#buildinlinestylesheets). Die Standardvoreinstellung war `"never"`.
+
+Astro v3.0 ändert den Standardwert von `inlineStylesheets` zu `"auto"`. Stylesheets kleiner als `ViteConfig.build.assetsInlineLimit` (Standard: 4 KB) werden standardmäßig eingefügt. Andernfalls werden Projekt-Styles in externen Stylesheets gesendet.
+
+#### Was soll ich tun?
+Wenn du das aktuelle Verhalten deines Projekts beibehalten möchtest, setze `build.inlineStylesheets` auf den vorherigen Standardwert `"never"`:
+
+
+```js title="astro.config.mjs" ins={4-6}
+import { defineConfig } from "astro/config";
+
+export default defineConfig({
+	 build: {
+    inlineStylesheets: "never"
+  }
+})
+```
+
+### Standardwert Geändert: Bilderdienst
+
+In Astro v2.x war Squoosh der [Standard-Bildverarbeitungsdienst](/de/guides/images/#default-image-service).
+
+Astro v3.0 enthält jetzt standardmäßig Sharp als Bildverarbeitungsdienst und bietet stattdessen eine Konfigurationsoption, um Squoosh zu verwenden.
+
+#### Was soll ich tun?
+
+:::note
+Bei Verwendung eines [strikten Paketmanagers](https://pnpm.io/pnpm-vs-npm#npms-flat-tree) wie `pnpm` musst du möglicherweise Sharp manuell in dein Projekt installieren, obwohl es eine Abhängigkeit von Astro ist:
+
+```bash
+pnpm install sharp
+```
+:::
+
+Wenn du Squoosh weiterhin für die Verarbeitung deiner Bilder verwenden möchtest, aktualisiere deine Konfiguration wie folgt:
+
+```ts title="astro.config.mjs" ins={4-6}
+import { defineConfig, squooshImageService } from "astro/config";
+
+export default defineConfig({
+  image: {
+    service: squooshImageService(),
+  }
+})
+```
+
+### Geändert: Groß- und Kleinschreibung der HTTP-Anforderungsmethoden
+
+In Astro v2.x wurden [HTTP-Anforderungsmethoden](/de/core-concepts/endpoints/#http-methods) mit Kleinbuchstaben verwendet: `get`, `post`, `put`, `all` und `del`.
```

Comment:
```suggestion
In Astro v2.x wurden [HTTP-Anforderungsmethoden](/de/core-concepts/endpoints/#http-methoden) mit Kleinbuchstaben verwendet: `get`, `post`, `put`, `all` und `del`.
```
Because the page is translated, we need to use the translated heading here!